CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing to systemlink-OpenAPI-documents 
+
+Contributions to systemlink-OpenAPI-documents are welcome from all!
+
+systemlink-OpenAPI-documents is managed via [git](https://git-scm.com), with the canonical upstream
+repository hosted on [GitHub](http://developercertificate.org/).
```

Comment:
Is this meant to be a link to GitHub instead of developercertificate.org?

Reply:
Yeah, I think so. That came from the NI template, so I'll try to get it fixed there as well.